package.json

```diff
@@ -3,12 +3,19 @@
   "description": "Realtime client library for Ably.io, the realtime messaging service",
   "version": "0.9.0-beta.0",
   "main": "./nodejs/index.js",
+  "react-native": {
+    "./nodejs/index.js": "browser/static/ably-reactnative.js"
+  },
+  "browser": {
+    "./nodejs/index.js": "browser/static/ably-commonjs.js"
+  },
   "dependencies": {
     "async": "git://github.com/ably-forks/async#requirejs",
     "hexy": "~0.2",
     "msgpack-js": "git://github.com/paddybyers/msgpack-js.git",
     "request": "^2.74.0",
-    "ws": "^1.1.1"
+    "ws": "^1.1.1",
+    "react-native-randombytes": "git+https://github.com/ably-forks/react-native-randombytes.git"
```

Comment:
This is actually used in runtime, and since it has native Obj-C and Java files we can't just concat the dependency on ably-reactnative.js. It's unfortunate that we force the dependency to every platform using this library, but as there's no way I know of of having platform-specific dependencies the only alternative I can think of is to have a separate `ably-react-native` library that wraps `ably-js` and provides the Platform object.